pphysx3d/src/shapes/bounding_volume.rs

```diff
@@ -33,8 +37,10 @@ impl BoundingVolume for AABB {
     fn contains(&self, other: &Self) -> bool {
         na::partial_le(&self.mins, &other.mins) && na::partial_ge(&self.maxs, &other.maxs)
```

Comment:
```suggestion
        na::partial_le(&self.mins, &other.mins()) && na::partial_ge(&self.maxs, &other.maxs())
```

Reply:
shouldnt it be be &self.mins() aswell?

Reply:
i don't think it matters if self uses the getter